### PR TITLE
66 front catch dns unicity error secure domain unicity

### PR DESCRIPTION
**Pull request by Trottoir**
Add Error management based on "error" events emitted

Error events have this format : `ERROR : The reason why the error occurs`

If the `Event Listener` trigger an error event, it'll stop the `stepper` process & display an alert with the Error content

### CODE REVIEW

File: pkg/front/errors.go [hunk 1]
```diff
@@ -1,28 +1,34 @@
...
-	['Wallet-2002', 'Error while trying to delete the wallet, may be opened by an other program'],
-	['Wallet-3001', 'Error while trying to import the wallet file'],
-	['Wallet-4001', 'Error while trying to fetch all your wallets'],
+	['Wallet-0001', 'That nickname is taken. Try Another'],
+	['Wallet-0002', 'Wrong password. Try again'],
+	['Wallet-0003', 'Error while retrieving that wallet. Try again'],
+	['Wallet-1001', 'Enter a wallet nickname'],
+	['Wallet-1002', 'Enter a wallet password'],
+	['Wallet-1003', 'Error while creating your wallet. Try again'],
+	['Wallet-2001', 'Select a wallet to delete’'],
```
Comment (Thykof) [inline pkg/front/errors.go:15]: typo here? backtick at the end of the message

Comment (Thykof) [inline pkg/front/errors.go:15]:
```suggestion
	['Wallet-2001', 'Select a wallet to delete'],
```

**Review by Thykof — APPROVED** [review]

**Review by SebastienValla — APPROVED** [review]

File: resource/html/front/wallet.html [hunk 2]
```diff
@@ -52,15 +52,20 @@ <h2>Thyra</h2>
...
+							>
 								Load a wallet
 							</button>
 						</div>
 					</div>
 
 					<div class="row">
 						<div id="create-form">
-							<h2 class="mb-4">Create a wallet</h2>
+							<h2 class="mb-4">Create Wallet</h2>
```
Comment (Thykof) [inline resource/html/front/wallet.html:68]: @SlnPons can you confirm we need 'Create Wallet'?

File: resource/html/front/website.js [hunk 3]
```diff
@@ -167,100 +175,77 @@ function tableInsert(resp, count) {
...
-
-		document.getElementsByClassName('fileTypeError')[0].style.display = 'flex';
-		document.getElementById('website-upload').style.display = 'none';
-		document.getElementById('website-upload-refuse').style.display = 'flex';
-	} else {
-		uploadable = true;
-		document.getElementsByClassName('fileTypeError')[0].style.display = 'none';
-		document.getElementById('website-upload').style.display = 'flex';
-		document.getElementById('website-upload-refuse').style.display = 'none';
-	}
```
Comment (Thykof) [inline resource/html/front/website.js:218]: why removing this check?

Comment (Thykof) [reply resource/html/front/website.js]: changed reverted, ok for me

**Review by Thykof — APPROVED** [review]

**Pull request merged**